packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/ModelController.java

```diff
@@ -378,65 +339,6 @@ ResponseEntity<Model> getModelFromConfigId(
 		}
 	}
 
-	@GetMapping("/search")
-	@Secured(Roles.USER)
-	@Operation(summary = "Search models with a query")
-	@ApiResponses(
-		value = {
-			@ApiResponse(
-				responseCode = "200",
-				description = "Models found.",
-				content = @Content(
-					array = @ArraySchema(schema = @io.swagger.v3.oas.annotations.media.Schema(implementation = Model.class))
-				)
-			),
-			@ApiResponse(
-				responseCode = "500",
-				description = "There was an issue retrieving models from the data store",
-				content = @Content
-			)
-		}
-	)
-	public ResponseEntity<List<Model>> searchModels(
```

Comment:
@kbirk do we want to strike this or will it still be useful for the new project-page searching?

Reply:
Nah that's good to nuke. The guts of it is in the service.